recipes-openxt/xenclient/xenclient-toolstack_git.bb

```diff
@@ -46,11 +46,16 @@ S = "${WORKDIR}/git"
 do_compile() {
         make V=1 XEN_DIST_ROOT="${STAGING_DIR}"
 }
-OCAML_INSTALL_LIBS  = "libs/uuid libs/stdext libs/mmap \
-                      libs/json libs/jsonrpc libs/http \
-                      libs/log libs/xc libs/eventchn \
-                      libs/xb libs/xs \
-		      libs/common"
+
+OCAML_INSTALL_LIBS = " \
+    libs/uuid \
+    libs/stdext \
+    libs/json \
+    libs/jsonrpc \
+    libs/http \
+    libs/log \
+    libs/common \
+    "
```

Comment:
The change here is deliberate. Those five libraries are no longer part of the toolstack build as the code has been removed. Upstream versions are packaged by the xen-libxl recipe instead and uid obtains copies of those libraries from the xen-ocaml-libs package which was introduced by the preceding commit in this PR -- that's why the uid recipe gains a DEPENDS on xen-libxl in this change.